app/src/main/java/com/kuit/ourmenu/utils/dragndrop/Ext.kt

```
@@ -0,0 +1,32 @@
+import androidx.compose.ui.composed
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.zIndex
+
+fun <T> MutableList<T>.move(from: Int, to: Int) {
+    if (from == to) return
+    if (to > this.size - 1) return
+    Log.d("DragAndDrop", "Moving item from $from to $to , ${this.size}")
+    val element = this.removeAt(from)
+    try {
+        this.add(to, element)
+    } catch (e: IndexOutOfBoundsException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)